src/test/java/ru/training/at/hw7/entities/User.java

```diff
@@ -0,0 +1,49 @@
+package ru.training.at.hw7.entities;
+
+import ru.training.at.hw7.testdata.TestDataLoader;
+
+import java.util.Properties;
+
+public class User {
+
+    private String name;
+    private String password;
+    private String fullName;
+
+    //Constructors
+    public User() {
+        System.out.println("Creating default User object");
+        Properties credentials = TestDataLoader.getCredentials();
+        this.name = credentials.getProperty("username");
+        this.password = credentials.getProperty("password");
+        this.fullName = "Roman Iovlev";
```

Comment:
I think `fullName` is a parameter that should be stored in same way as `name/password`

Reply:
Fixed. Now it is in userdata.properties file